### ISSUE TITLE
'.' is no longer in @INC

### ISSUE THREAD

**Issue by eserte** [Grinnz/Mojo-Reactor-Epoll#2]
Newer perl versions don't have anymore "." in `@INC`. This causes failures in the test suite:
```
do "t/00-report-prereqs.dd" failed, '.' is no longer in @INC; did you mean do "./t/00-report-prereqs.dd"? at t/00-report-prereqs.t line 71.
```
Current CPAN.pm and other installers workaround the issue, so this problem is not visible when using these tools. But it fails if the distribution is built manually on the commandline.

A possible fix is to put something like `use lib ".";` on top of the failing test scripts.

**Comment by Grinnz** [comment]
This is due to test files generated by an old version of [Test::ReportPrereqs]. I will make a new release. Thanks!

**Comment by Grinnz** [comment]
This should be fixed by 0.007
